frontend/lib/src/components/shared/Icon/DynamicIcon.tsx

```diff
@@ -39,10 +39,16 @@ function parseIconPackEntry(iconName: string): IconPackEntry {
   return { pack: iconPack, icon: iconNameInPack }
 }
 
-export function isMaterialIcon(option: string): boolean {
-  const materialIconRegexp = /^:material\/(.+):$/
-  const materialIconMatch = materialIconRegexp.exec(option)
-  return materialIconMatch !== null
+export function isFilledStarIcon(content: string): boolean {
```

Comment:
nit: since it's only used in feedback/button group, it might also be fine to move that to button group... but I have no strong opinion

Reply:
that's a good point, but since we are likely continue to work on button group and figure out the details then, I would suggest to make the changes then 👍